helpers/calendar/credentials.py

```diff
@@ -34,3 +34,28 @@ def set_api_credentials(self):
         service = build('calendar', 'v3', developerKey=api_key,
                         http=credentials.authorize(Http()))
         return service
+
+
+def get_events_within_datetime_range(calendarId=None, timeMin=None,
```

Comment:
Since the difference between this function and the function defined in line 50 is the arguments passed to them, would it be appropriate to have only one function defined for both of them?

Reply:
Thanks for pointing this repetition out. It has been worked on.